sign-in.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <title>SpaceWalkers</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Russo+One|Ubuntu" rel="stylesheet">
+  <link href="css/index.css" rel="stylesheet">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+<body>
+  <header>
+    <h1>SpaceWalker</h1>
+
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="#">About Us</a>
+      <a href="#">Blog</a>
+      <a href="#">Contact</a>
+      <a href="sign-in.html">Sign In</a>
+    </nav>
+  </header>
+
+  <div class="sign-in-container">
+    <img class="contact-jumbo-top-img" src="img/space-walk.jpg" alt="Space view of Earth">
+    <form action="#">
+      <div class="sign-in-container sign-up-form">
```

Comment:
This entire HTML page is very well written. In particular, I love the naming scheme you've used for your classes.